scripts/configure-dev.sh

```diff
@@ -0,0 +1,77 @@
+replace_values BASE64_ENCODED_SSL_CERTIFICATE_TLS_CRT $BASE64_ENCODED_SSL_CERTIFICATE_TLS_CRT && \
+replace_values BASE64_ENCODED_SSL_CERTIFICATE_TLS_KEY $BASE64_ENCODED_SSL_CERTIFICATE_TLS_KEY && \
+replace_values STAKATER_PLATFORM_SSH_GIT_URL $STAKATER_PLATFORM_SSH_GIT_URL && \
+replace_values STAKATER_PLATFORM_BRANCH $STAKATER_PLATFORM_BRANCH && \
+replace_values USER_MAIL $USER_MAIL && \
+replace_values USER_NAME $USER_NAME && \
+replace_values REPO_ACCESS_TOKEN $REPO_ACCESS_TOKEN && \
+replace_values EXTERNAL_DNS_AWS_ACCESS_KEY_ID $EXTERNAL_DNS_AWS_ACCESS_KEY_ID ENCODE && \
+replace_values EXTERNAL_DNS_AWS_SECRET_ACCESS_KEY $EXTERNAL_DNS_AWS_SECRET_ACCESS_KEY ENCODE && \
+replace_values IMC_API_KEY $IMC_ALERT_CONTACTS && \
+replace_values NEXUS_ADMIN_ACCOUNT_USER $NEXUS_ADMIN_ACCOUNT_PASSWORD && \
+replace_values NEXUS_CLUSTER_ACCOUNT_USER $NEXUS_CLUSTER_ACCOUNT_PASSWORD && \
+replace_values KEYCLOAK_CLIENT_ID $KEYCLOAK_CLIENT_ID && \
+replace_values KEYCLOAK_CLIENT_SECRET $KEYCLOAK_CLIENT_SECRET && \
+replace_values KEYCLOAK_DEFAULT_PASSWORD $KEYCLOAK_DEFAULT_PASSWORD && \
+replace_values KEYCLOAK_DEFAULT_USERNAME $KEYCLOAK_DEFAULT_USERNAME && \
+replace_values KEYCLOAK_DB_USER $KEYCLOAK_DB_USER && \
+replace_values KEYCLOAK_DB_PASSWORD $KEYCLOAK_DB_PASSWORD && \
+replace_values JENKINS_NOTIFICATIONS_SLACK_CHANNEL $JENKINS_NOTIFICATIONS_SLACK_CHANNEL && \
+replace_values JENKINS_NOTIFICATIONS_SLACK_WEBHOOK_URL $JENKINS_NOTIFICATIONS_SLACK_WEBHOOK_URL && \
+replace_values JENKINS_PIPELINE_GITHUB_TOKEN $JENKINS_PIPELINE_GITHUB_TOKEN && \
+replace_values JENKINS_PIPELINE_GITLAB_TOKEN $JENKINS_PIPELINE_GITLAB_TOKEN && \
+replace_values JENKINS_PIPELINE_BITBUCKET_TOKEN $JENKINS_PIPELINE_BITBUCKET_TOKEN && \
+replace_values JENKINS_DOCKER_RELEASE_USERNAME $JENKINS_DOCKER_RELEASE_USERNAME && \
+replace_values JENKINS_DOCKER_RELEASE_PASSWORD $JENKINS_DOCKER_RELEASE_PASSWORD && \
+replace_values JENKINS_LOCAL_NEXUS_USERNAME $JENKINS_LOCAL_NEXUS_USERNAME && \
+replace_values JENKINS_LOCAL_NEXUS_PASSWORD $JENKINS_LOCAL_NEXUS_PASSWORD && \
+replace_values JENKINS_NEXUS_USERNAME $JENKINS_NEXUS_USERNAME && \
+replace_values JENKINS_NEXUS_PASSWORD $JENKINS_NEXUS_PASSWORD && \
+replace_values SLACK_INFRA_ALERTS_WEBHOOK_URL $SLACK_INFRA_ALERTS_WEBHOOK_URL && \
+replace_values SLACK_INFRA_ALERTS_CHANNEL $SLACK_INFRA_ALERTS_CHANNEL && \
+replace_values SLACK_APPS_ALERTS_WEBHOOK_URL $SLACK_APPS_ALERTS_WEBHOOK_URL && \
+replace_values SLACK_APPS_ALERTS_CHANNEL $SLACK_APPS_ALERTS_CHANNEL && \
+replace_values GRAFANA_USERNAME $GRAFANA_USERNAME && \
+replace_values GRAFANA_PASSWORD $GRAFANA_PASSWORD
+
+# Replace following Configs with their base64 encoded values in secrets in platform
+replace_configs  BASE64_ENCODED_ALERTMANAGER_CONFIG ../configs/alertmanager.yaml && \
+replace_configs  BASE64_ENCODED_IMC_CONFIG ../configs/imc.yaml && \
+replace_configs  BASE64_ENCODED_JENKINS_CFG ../configs/jenkins.json && \
```

Comment:
Convention over configuration CFG should be changed to CONFIG

Reply:
Ignore makefile-ci for now. We'll merge all of this into makefile eventually to keep things consistent